packages/trace-viewer/src/ui/uiModeFiltersView.tsx

```diff
@@ -80,9 +79,8 @@ export const FiltersView: React.FC<{
                 const copy = new Map(projectFilters);
                 copy.set(projectName, !copy.get(projectName));
                 setProjectFilters(copy);
-                const configFile = testModel?.config?.configFile;
```

Comment:
It is safe to use testModel.config.configFile - it will always be there in the UI mode scenario.